File: MediATech/Model/BL/Bluray.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MediATech.Model.BL
{
    public class Bluray : MediaObject
    {
        private bool _is4K;
        private int _duration;


        public Bluray(int id, string title, bool available, string state, DateOnly publicationDate, bool is4K, int duration)
            : base(id, title, available, state, publicationDate)
        {

        }


        public bool Is4K { get => _is4K; set => _is4K = value; }

        public int Duration { get => _duration; set => _duration = value; }
    }
}

File: src/main/java/mediatech/Model/BL/Bluray.java
package mediatech.Model.BL;

import java.util.Date;

public class Bluray extends MediaObject {

    private boolean is4K;
    private int duration;

    public Bluray(int id, String title, boolean available, String state, Date publicationDate, boolean is4K, int duration) {
        super(id, title, available, state, publicationDate);

        this.is4K = is4K;
        this.duration = duration;
    }

    public Bluray() {}

    public boolean getIs4K() {
        return is4K;
    }
    public void setIs4K(boolean is4K) {
        this.is4K = is4K;
    }

    public int getDuration() {
        return duration;
    }
    public void setDuration(int duration) {
        this.duration = duration;
    }
}
